sentinel/src/lib/feed.js

```diff
@@ -1,16 +1,32 @@
+const async = require('async');
 const logger = require('./logger');
 const db = require('../db');
 const metadata = require('./metadata');
 const tombstoneUtils = require('@medic/tombstone-utils');
+const transitionsLib = require('../config').getTransitionsLib();
 
 // we don't run transitions on ddocs or info docs
 const IDS_TO_IGNORE = /^_design\/|-info$/;
 
 const RETRY_TIMEOUT = 60000; // 1 minute
+const PROGRESS_REPORT_INTERVAL = 500; // items
+const MAX_QUEUE_SIZE = 100;
 
-let listener;
 let init;
```

Comment:
What information does `init` expose that `request` doesn't? If `request` is undefined or cancelled that is the same as init being null right? Could these two be combined?